audio/beets/Portfile

```diff
@@ -0,0 +1,517 @@
+
+    depends_lib-append \
+                    port:py${python.version}-alive-progress
+}
+
+subport beets-ibroadcast {
+    name            beets-ibroadcast
+    version         0.5.0
+    revision        0
+
+    license         Unlicense
+
+    description     iBroadcast plugin for Beets
+    long_description ${description}
+
+    homepage        https://github.com/ctrueden/beets-ibroadcast
+
+    checksums       rmd160  bbd7ede95fa6d20b55c9a86b9dc540a1b9bd3b3a \
+                    sha256  838968ee4ccb3d08dbb8c465efbbcf0f2c87b387402e91fa1b506c01b0775db2 \
+                    size    10974
+
+    depends_lib-append \
+                    port:py${python.version}-alive-progress
+}
+
+subport beets-importreplace {
+    PortGroup       github 1.0
+
+    github.setup    edgars-supe beets-importreplace 0.1.1 v
+    revision        0
+
+    license         MIT
+
+    description     Plugin for beets to perform regex replacements during import
+    long_description ${description}
+
+    homepage        https://github.com/edgars-supe/beets-importreplace
+
+    checksums       rmd160  36ae8f85b72aa291dcd429967abb772faa05a066 \
+                    sha256  8c8466b57a75a0e6be9d9f5daf1d6e322e7ee8abecb4df665c818f48b3b8df67 \
+                    size    4115
+}
+
+subport beets-mosaic {
+    name            beets-mosaic
+    python.rootname beets_mosaic
+    version         0.9.1
+    revision        0
+
+    license         MIT
+
+    description     Plugin for the music library manager Beets. The mosaic plugin generates a montage of a mosiac from cover art.
+    long_description ${description}
+
+    homepage        https://github.com/susannamaria/beets-mosaic
+
+    checksums       rmd160  71d68c17894f255fcea01d6da07782e8eab9b324 \
+                    sha256  7e6ff14be69502c394dfc7abe57f144747adba700a2a50dd36c06e9a3e18a966 \
+                    size    6195
+
+    depends_lib-append \
+                    port:py${python.version}-fonttools \
+                    port:py${python.version}-parse \
+                    port:py${python.version}-pillow \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-ttfquery
+}
+
+subport beets-noimport {
+    name            beets-noimport
+    version         0.1.0b5
+    revision        0
+
+    license         MIT
+
+    description     Beets plugin to add directories to the incremental import 'do not import' list
+    long_description ${description}
+
+    homepage        http://www.github.com/ttsda/beets-noimport
+
+    checksums       rmd160  2755bf7b969712da0931c5972ddf23f6aff7ec90 \
+                    sha256  86a6291e1de90b292829e6f7286b057be2bc556f5a49e9c30ffead2985ae998a \
+                    size    3167
+}
+
+subport beets-originquery {
+    PortGroup       github 1.0
+
+    github.setup    x1ppy beets-originquery 269e9e4cfc88128493b25ed814e34df2528dc1ef
+    version         20210908
+    revision        0
+
+    license         Unknown
```

Comment:
It's not clear indeed what's the intended license - you could ask upstream about it